client/Cargo.toml

```
@@ -0,0 +1,14 @@
+[package]
+name = "fuel-debugger-client"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+clap = { version = "3.1", features = ["env", "derive"] }
+fuel-debugger = { path = "../", version = "0.1" }
```

This unfortunately needs to be an exact version for the publishing workflow to work properly 😞
```suggestion
fuel-debugger = { path = "../", version = "0.1.0" }
```